lib/ain-ocean/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod storage;
 use crate::storage::ocean_store::OceanStore;
 
 pub type Result<T> = std::result::Result<T, Error>;
+pub type ApiResult<T> = std::result::Result<T, ApiError>;
```

Comment:
hmm... why do we need this?? is this unused??

Reply:
this need to remove